.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "parser": "babel-eslint",
+  "extends": [
+    "standard",
+    "standard-react"
+  ],
+  "env": {
+    "browser": true,
+    "node": true,
+    "jest": true
+  },
+  "rules": {
+    // Temporary, components need to be rewritten to take advantage of this
```

Comment:
Note that the eslint parser is happy with comments (even though this is a JSON file).

Temporary comment anyway as mentioned in the MR description.